geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -17,6 +17,10 @@ Changes to apply
 1. The configuration vars `vars/functionalities/anonymous` and `vars/functionalities/registered` should
    be moved to the new roles `anonymous` and `registered` that will exist after the upgrade of the database.
 
+2. The 'INSTANCE' configuration variable is removed, it should be in the '.env' files, and also the
+   environment makefiles that also sould be moved in the '.env' files, in a multiporgect you can have a
```

Comment:
"environment makefiles, these contents should also be moved to the '.env' files. In a multi-instance project you can have a